npm-shrinkwrap.json

```diff
@@ -1242,9 +1242,9 @@
       "dev": true
     },
     "coffee-script": {
-      "version": "1.11.1",
+      "version": "1.12.0",
```

Comment:
Looks like this might have actually broke the build as `for..from` is [a new CS feature](http://coffeescript.org/#changelog).